cla_public/templates/base.html

```diff
@@ -54,15 +54,9 @@
     {% endif %}
   {% endif %}
   <div class="laa-flee-button-container
-    {% if ('n43n3' in session.checker.diagnosis_previous_choices) or
-        ('n97' in session.checker.diagnosis_previous_choices) or
-        ('n86' in session.checker.diagnosis_previous_choices) or
-        ('n149' in session.checker.diagnosis_previous_choices) or
-        ('n18' in session.checker.diagnosis_previous_choices) or
-        ('n19' in session.checker.diagnosis_previous_choices) or
-        ('n88' in session.checker.diagnosis_previous_choices)
-    %}
+    {% if is_quick_exit_enabled(node, session) %}
       laa-fee-button-enabled
```

Comment:
I know this isn't new in this PR but this looks like a typo of 'flee'. If that's the case, do you think it's in scope to fix in this PR? If it isn't, it might need a comment.

Reply:
I shall check and change - I can do it as part of this PR.